pcdsdevices/ek9000.py

```diff
@@ -0,0 +1,39 @@
+"""
+Classes for cards attached to EK9000 bus couplers.
+"""
+
+from ophyd import Component as Cpt
+from ophyd import Device
+from ophyd.signal import EpicsSignal, EpicsSignalRO
+
+
+class EL3174_AI_Ch(Device):
```

Comment:
The Python convention for class names from PEP-8 is CamelCase without underscores.